modules/grafana/templates/dashboards/publishing_api_overview.json.erb

```diff
@@ -829,6 +829,91 @@
         }
       ],
       "title": "New row"
+    },
+    {
+      "collapse": false,
+      "editable": true,
+      "height": "250px",
+      "panels": [
+        {
+          "aliasColors": {},
+          "bars": false,
+          "datasource": "Graphite",
+          "editable": true,
+          "error": false,
+          "fill": 1,
+          "grid": {
+            "threshold1": null,
```

Comment:
I believe the desired values are already exported through the icinga::check::graphite resources [1] as the warning and critical attributes, but... I'm still no closer to a solution, I just can't work out what "realize"'ing them in the s_redis_base class does, and how that helps getting closer to accessing the value of the attributes... puppet is making me sad :(

1: https://github.com/alphagov/govuk-puppet/blob/86350f0a6e2ebc9e4be9c1df96e24739b63d5ddf/modules/govuk/manifests/node/s_redis_base.pp#L20